src/components/related/OutfitList.jsx

```diff
@@ -36,21 +38,26 @@ const OutfitList = () => {
     getStoredOutfit()
   }
 
+
   return (
-    <div className={styles.OutfitList}>
-      Add to Outfit
-      <img
-        src={plusSymbol}
-        style={{ height: '200px', width: '200px'}}
-        onClick={handleAddOutfit}
-      />
-      {Object.values(outfitData).map((storedOutfit) => {
-        return <OutfitCard
-          key={storedOutfit.id}
-          storedOutfit={storedOutfit}
-          getStoredOutfit={getStoredOutfit}
+    <div className='outfit-list'>
+      <div className='add'>
+        <img
+          className='img'
+          src={plusSymbol}
+          style={style}
```

Comment:
I like how you moved the style out to a separate variable, I think it definitely helps make it more readable. Potentially you could move this to the img class though? Or are there other images that use the class and need to be a different size?

Reply:
yeah great suggestion, I will definitely do that on my next PR.

Reply:
Unfortunately the styling does not when moving into the img class